### DISCUSSION TITLE
Php editor v2 problem

### DISCUSSION THREAD

**Discussion by symio** [junichi11/netbeans-wordpress-plugin#76] [Q&A] [answered]
Hello Junichi,

Like in your other plugin PHP enhancement I encounter the "PHP editor v2" problem.
I tried to install netbeans-wordpress from then embed plugin center as by downloading the nbm and using the downloaded tab but none works.
I use Nb 12.4, have you any tip or workaround for this ? 
Maybe by recomiling the plugin after changing a little string somewhere ?
I like your work and have to develop 2 wp plugins and it would have been better for me with your plugin ...

Thank you and best regards,

**Comment by symio** [comment, marked as answer]
I did'nt view the 1.2 tag just thank you and sorry for the false alert, my bad

**Reply by junichi11** [reply]
No problem :) We can't upload the new version to the PP because the PP doesn't support feature release version numbers. Sorry for the inconvenience.

**Reply by symio** [reply]
Yes, I understood it in your previous reply and nb oppened issue.

Thank you so much for your contributions.
